css/index.css

```diff
@@ -1,3 +1,292 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+ol, ul {
+	list-style: circle;
+}
+
+.main-content {
+    position: absolute;
+    margin: auto;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+	max-width: 900px;
+}
+
+header {
+    display: flex;
+    width: 100%;
+    margin: 20px 0;
+}
+
+nav {
+	display: flex;
+	justify-content: flex-start;
+    width: 100%;
+    
+}
+
+nav a {
+    margin: 0 80px 0 0;
+	margin-top: 1%;
+	text-decoration: none;
+	color: grey;
+}
+
+
+.top-feature {
+	display: flex;
+	justify-content: space-evenly;
+	margin: 50px 0 ;
+}
+
+.feature-text {
+	width: 30%;
+	height: 220px;
+	margin: 5% 10%;
+}
+
+.feature-text h2 {
+	font-size: 70px;
+	display: flex;
+	justify-content: center;
+}
+
+.get-started-btn {
+    margin-left: 30px;
+	width: 200px;
+	height: 35px;
+	background-color: #fff;
+}
+
+.top-content {
+	display: flex;
+	margin: 4% 0;
+	justify-content: space-around;
+}
+
+.features {
+	width: 45%;
+}
+
+.about {
+	width: 45%;
+}
+
+.middle-image {
+	display: flex;
+	justify-content: center;
+}
+
+.bottom-content {
+	display: flex;
+	justify-content: space-around;
+	margin: 4% 0;
+}
+
+.services {
+	width: 30%;
+}
+
+.product {
+	width: 30%;
+}
+
+.vision {
+	width: 30%;
+}
+
+.main-address {
+	display: flex;
+	flex-direction: column;
+}
+
+.main-phone {
+	margin: 2% 0;
+}
+
+.copyright {
+	display: flex;
+	justify-content: center;
+}
+
+/* Tablet View */
+
+@media (min-width: 768px) and (max-width: 1024px) {
```

Comment:
This is a smart idea, at least to me. You can avoid a lot of specificity and cascading issues by making this section of CSS to be applied only between 768px and 1024px.

By the way, the assignment says the breakpoints are (_max_width_) 800px and 500px, but I think I see why you opted for 1024px and 768px instead.